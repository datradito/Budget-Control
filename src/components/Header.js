import React from 'react';
import {  NavLink } from 'react-router-dom';


const Header = () => (
    <header>
        <h1>Control de Gastos Giuliana</h1>
        <NavLink to="/" exact={true} activeClassName="is-active">Pantalla Principal</NavLink>
        <NavLink to="/crear" activeClassName="is-active">Add GastosPage Test</NavLink>
        <NavLink to="/help" activeClassName="is-active">Help!</NavLink>
    </header>
)

export default Header;