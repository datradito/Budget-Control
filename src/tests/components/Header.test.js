import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../components/Header';


test('Deberia render Header correctamente', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
})