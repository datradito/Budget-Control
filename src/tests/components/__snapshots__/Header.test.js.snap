// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Deberia render Header correctamente 1`] = `
<header>
  <h1>
    Control de Gastos Giuliana
  </h1>
  <NavLink
    activeClassName="is-active"
    aria-current="page"
    exact={true}
    to="/"
  >
    Pantalla Principal
  </NavLink>
  <NavLink
    activeClassName="is-active"
    aria-current="page"
    to="/crear"
  >
    Add GastosPage Test
  </NavLink>
  <NavLink
    activeClassName="is-active"
    aria-current="page"
    to="/help"
  >
    Help!
  </NavLink>
</header>
`;
